.github/ISSUE_TEMPLATE/bug-template.md

```diff
@@ -1,27 +0,0 @@
----
```

Comment:
Can we readd the existing templates with zenhub prefix so people can still use markdown if they'd like and that they can be used on zenhub for those who like to create issues there please?